### ISSUE TITLE
Project support

### ISSUE THREAD

**Issue by chris-umed** [django-helpdesk/django-helpdesk#1023]
@gwasser @reduxionist  - would you consider adding more admins to your project to improve the throughput and and core testing functionality to enforce standardisation that should help in overall quality of the project?

**Comment by gwasser** [comment]
Yes of course, help is welcome for PRs and would definitely consider adding more admins that become familiar with the project. I have not had the time lately to do as much as I hoped, so appreciate folks stepping up to help with that.

**Comment by chris-umed** [comment]
Excellent - please add these user ID's to your project and we will get cracking bringing the project up to scratch:
uhurusurfa
superuser-marty-me-uk

**Comment by gwasser** [comment]
Could you start with a PR please @chris-umed? It doesn't appear you've contributed to the project before so I just want to make sure we're on the same page with what kind of changes are needed. Thanks.

**Comment by gwasser** [comment]
Sounds great @chris-umed. pycodestyle works for me. We were using pycodestyle back when travis ci was free but I'm not sure if it's the azure pipeline anymore.

**Comment by gwasser** [comment]
Since this is kind of a help wanted thread now, it'd be great if we could include some kind of anti-spam tech to the public facing interface. Our demo site gets flooded with spam posts and I have to clear them out regularly, it'd be great if we could stop or at least slow that if anyone feels they can tackle it. I think it would ideally be an in-house solution not reliant on google captcha or any third party website.

**Comment by nickmccullum** [comment]
@gwasser I would also be interested in signing up as a maintainer of this project. We use it at large scale in a production application, and I contributed the PR to upgrade the project to Django 4 functionality.

**Comment by gwasser** [comment]
For sure @nickmccullum, appreciate the help! @chris-umed have you worked on a PR recently? I'll have to look at setting more folks as maintainers. Ideally we'd come up with a plan together to put in new features and address some of the ticket backlog. Thank you!

**Comment by uhurusurfa** [comment]
I am "chris-umed" from above. @martin-marty is working with me and his efforts are funded by my company.
If we are to move this project forward then there needs to be some key changes made immediately.
The biggest one is that there can be NO MORE PR merges that have failing unit tests. This is a fundamental development paradigm for quality software.

The unit tests were fixed where we could easily ddo that or disabled where the unit test clearly need a complete rewrite in this PR:


However, the subsequent PR then broke it again and was merged with failures:
https://github.com/django-helpdesk/django-helpdesk/pull/1024

Let me know ASSAP if you are willing to enable prevention of merges with CI/CD failures please.

**Comment by uhurusurfa** [comment]
Ok - could you explain the git flow paradigm you are following for development?
Without a clear understanding of how you are managing branches, this could create more work.

**Comment by timthelion** [comment]
@uhurusurfa I did skim the thread but I'm still confused. Are you planning on paying gwasser?

**Comment by uhurusurfa** [comment]
@timthelion Who exactly are you and what is your stake in this conversation?

**Comment by timthelion** [comment]
https://github.com/django-helpdesk/django-helpdesk/graphs/contributors

**Comment by uhurusurfa** [comment]
@timthelion Wonderful

**Comment by gwasser** [comment]
@uhurusurfa in terms of gitflow, the idea is that `stable` is our main release branch. Versions are branched off from `stable` as they are retired so its easy to find past releases if necessary (in theory could even develop emergency bugfixes). Our development branch is `unstable` that gets merged into `stable` when new releases are ready. New features should generally get developed on branches off of `unstable` then get merged in after review and testing.

This was the idea. This project originally had no real process, so I've been slowly moving toward it. Granted, not always perfect at following since it's still a relatively small project with few contributors, but I support making it more "serious" to help the project grow.

**Comment by Mte90** [comment]
Thanks I got an upgrade as Triage in this repo 😁
I contributed because it is our plan to migrate to this software for our tracking system.

Maybe one of the things I can do is to review the PR (https://github.com/django-helpdesk/django-helpdesk/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc) still open after years (if I got some time) just to cleaning up.

Another idea is to give priorities to tickets marked as bug (https://github.com/django-helpdesk/django-helpdesk/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug) that are 40 to review them and assign, so we can be free during our weekends and sleep better 🤪

**Comment by gwasser** [comment]
Thank you @Mte90! Any help reviewing the ticket backlog and PRs is most appreciated! We should sort them based on priority and features. Getting our unit testing right is important so we can test fixes and features later.
